package/gluon-web-osm/luasrc/usr/lib/lua/gluon/web/model/osm.lua

```diff
@@ -4,7 +4,8 @@ local util = require "gluon.web.util"
 local class = util.class
 
 
-local DEFAULT_URL = 'https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.2.0'
+local DEFAULT_URL =
+	'https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@35ffe7626ce16c372143f3c903950750075e7068/en/v5.3.0'
```

Comment:
```
❯ http head https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@35ffe7626ce16c372143f3c903950750075e7068/en/v5.3.0
HTTP/1.1 404 Not Found
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: *
CF-Cache-Status: BYPASS
CF-RAY: 6dd10af1ed9f5c02-FRA
Cache-Control: no-cache, no-store, must-revalidate
Connection: keep-alive
Content-Encoding: br
Content-Type: text/plain; charset=utf-8
Cross-Origin-Resource-Policy: cross-origin
Date: Sun, 13 Feb 2022 21:10:19 GMT
ETag: W/"4f-tTE7/6pMHYHxorpZr3xhy03GBFc"
Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Timing-Allow-Origin: *
Vary: Accept-Encoding
X-Cache: HIT
X-Content-Type-Options: nosniff
X-Served-By: cache-fra19153-FRA
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400
```

Reply:
The base URL only refers to the directory, the actual referenced files are:
- https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@35ffe7626ce16c372143f3c903950750075e7068/en/v5.3.0/build/ol.js
- https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@35ffe7626ce16c372143f3c903950750075e7068/en/v5.3.0/css/ol.css

Reply:
Would've expected the variable to be called DEFAULT_BASE_URL then.

Reply:
Well, it is the default for the site.conf `openlayers_url` option. I guess something like `openlayers_base_url` would be more fitting, but I don't think it's important enough to justify changing it.